projects/core/components/primitive-textfield/primitive-textfield-options.ts

```diff
@@ -1,9 +1,7 @@
 import {InjectionToken, ValueProvider} from '@angular/core';
```

Comment:
This file is now a bit ridiculous 😄

Reply:
yeah 😄 but hope it can be useful for something else in the future